packages/extension-commerce-bm-seo/jest.config.js

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2025, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const base = require('@salesforce/pwa-kit-dev/configs/jest/jest.config.js')
+
+module.exports = {
+    ...base,
+    moduleNameMapper: {
+        ...base.moduleNameMapper,
+
+        // handle pwa-kit extensibility special import
+        '^overridable!(.*)': '$1'
```

Comment:
No action required, but we might want to move this somewhere reusable so it doesn't show up in all the extensions. A simple solution would be to have a special config like '@salesforce/pwa-kit-dev/configs/jest/jest.config-extension.js')